dev/datasets/gammapy-data-index.json

```diff
@@ -7,55 +7,64 @@
     "path": "cta-1dc/README.md",
     "url": "https://github.com/gammapy/gammapy-extra/raw/master/datasets/cta-1dc/README.md",
     "filesize": 497,
-    "hashmd5": "64cde583f9a4f2c655d008a0c6a69da3"
+    "hashmd5": "64cde583f9a4f2c655d008a0c6a69da3",
+    "itempath": "/Users/jer/git/gammapy/gammapy-extra/datasets/cta-1dc/README.md"
```

Comment:
Why is this local path needed? I think we should clearly avoid writing user specific path information for the index file...do you have any idea how to avoid this?

Reply:
You're right.
I've made a commit with the modif that removes this local path from the index file.